dkc/core/admin/folder.py

```diff
@@ -9,3 +11,15 @@ class FolderAdmin(admin.ModelAdmin):
     list_display_links = ['id', 'name']
 
     search_fields = ['name', 'id']
+
+    @transaction.atomic
+    def save_model(self, request, obj: Folder, form, change: bool):
+        if not change:
+            if obj.parent:
+                obj.tree = obj.parent.tree
+            else:
+                obj.tree = Tree.objects.create(quota=request.user.quota)
```

Comment:
What if a tree is already selected?

Reply:
What do you mean already selected? Tree is always a computed property, it can't be manually selected even from the admin page.

Reply:
Oh, I see it's `editable=False`. I thought we had a little more discretion to set this up manually in the admin.